lib/exercism/rikki.rb

```diff
@@ -1,19 +1,27 @@
 require 'digest/sha1'
 
-class Rikki
-  def self.validate(key)
+module Rikki
+  extend self
```

Comment:
rikki is singular, think of it as a bot. If it does more than one thing, it will be managed on the bot side of things, not the site side of things.